agents/astagent.go

```diff
@@ -86,7 +86,7 @@ func (sma *AsteriskAgent) connectAsterisk(stopChan <-chan struct{}) (err error)
 	sma.astConn, err = aringo.NewARInGO(fmt.Sprintf("ws://%s/ari/events?api_key=%s:%s&app=%s",
 		connCfg.Address, connCfg.User, connCfg.Password, CGRAuthAPP), "http://cgrates.org",
 		connCfg.User, connCfg.Password, fmt.Sprintf("%s@%s", utils.CGRateS, utils.Version),
-		sma.astEvChan, sma.astErrChan, stopChan, connCfg.ConnectAttempts, connCfg.Reconnects)
+		sma.astEvChan, sma.astErrChan, stopChan, connCfg.ConnectAttempts, connCfg.Reconnects, connCfg.MaxReconnectInterval, utils.FibDuration)
```

Comment:
Please try to make the function signature without the MaxReconnectInterval since I have the feeling it should be included within FibDuration function. This should be for all the libaries.